src/VisualStudio/IntegrationTests/AbstractInteractiveWindowTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.VisualStudio.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.OutOfProcess;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests
+{
+    public abstract class AbstractInteractiveWindowTest : AbstractIntegrationTest
+    {
+        private const string Edit_SelectionCancelCommand = "Edit.SelectionCancel";
+
+        private static readonly char[] LineSeparators = { '\r', '\n' };
```

Comment:
>private static readonly char[] LineSeparators = { '\r', '\n' }; [](start = 8, length = 63)

This is not the complete set of line separators that Visual Studio recognizes.  In that okay for this test?